canonical-web-and-design/jaas-dashboard
# Add Vanilla table sorting to Model list and Details pages

## Done

Added Vanilla table header sorting to the tables on the Model list and Details pages.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Try sorting the tables in all of the groupings on the model listing page, note that it still works with filters.
- Try sorting the tables in the model details pages.

## Details

Fixes #570
